test/linalg/test_linalg.fypp

```diff
@@ -48,6 +48,16 @@ contains
             new_unittest("trace_int16", test_trace_int16), &
             new_unittest("trace_int32", test_trace_int32), &
             new_unittest("trace_int64", test_trace_int64), &
+            new_unittest("kronecker_product_rsp", test_kronecker_product_rsp), &
+            new_unittest("kronecker_product_rdp", test_kronecker_product_rdp), &
+            new_unittest("kronecker_product_rqp", test_kronecker_product_rqp), &
+            new_unittest("kronecker_product_csp", test_kronecker_product_csp), &
+            new_unittest("kronecker_product_cdp", test_kronecker_product_cdp), &
+            new_unittest("kronecker_product_cqp", test_kronecker_product_cqp), &
+            new_unittest("kronecker_product_int8", test_kronecker_product_iint8), &
+            new_unittest("kronecker_product_int16", test_kronecker_product_iint16), &
+            new_unittest("kronecker_product_int32", test_kronecker_product_iint32), &
+            new_unittest("kronecker_product_int64", test_kronecker_product_iint64), &
```

Comment:
All of these should be generated by  `fypp` too. Otherwise there might be issues with `qp` precision (i.e. if it is supported or not)

Reply:
Alright. I made the change, and the build built and all tests passed. I checked the built `build/test/test_linalg.f90` and it seemed to have them listed. I suppose we could really clean up the linalg test source if all the other linalg tests were fypp generated like this.